exe-unit/src/network/inet.rs

```diff
@@ -516,7 +514,6 @@ async fn inet_tcp_proxy<'a>(ip: IpAddr, port: u16) -> Result<(TransportSender, T
     log::debug!("[inet] connecting TCP to {}:{}", ip, port);
 
     let tcp_stream = TcpStream::connect((ip, port)).await?;
-    let _ = tcp_stream.set_keepalive(Some(TCP_KEEP_ALIVE));
```

Comment:
@mfranciszkiewicz I had to remove this line, because new tokio::TcpStream doesn't have equivalent. is this dangerous?

Reply:
Connections may die due to lack of activity. We can use a workaround similar to the one implemented in hyper
https://github.com/hyperium/hyper/blob/055b4e7ea6bd22859c20d60776b0c8f20d27498e/src/client/connect/http.rs#L588-L673